src/app/search/search.component.ts

```diff
@@ -214,12 +214,11 @@ export class SearchComponent
     }
 
     public onSelectDataset(data: { ownerName: string; id: string }): void {
-        const id: string = data.id;
-        this.navigationService.navigateToDatasetView(
-            data.ownerName,
-            id,
-            ProjectLinks.urlDatasetViewOverviewType,
-        );
+        this.navigationService.navigateToDatasetView({
+            accountName: AppValues.defaultUsername,
+            datasetName: data.ownerName,
+            tab: ProjectLinks.urlDatasetViewOverviewType,
+        });
```

Comment:
The parameters look completely messed up.
`accountName` should be assigned to `data.ownerName`.
`datasetName` should be assigned to `data.name` (you need to pass it)

Reply:
Fixed